xml/System.Reflection/Assembly.xml

```diff
@@ -5041,7 +5081,17 @@ The ability to execute code in remote assemblies is disabled. See <see href="htt
         <param name="hashAlgorithm">The hash algorithm used for hashing files and for generating the strong name.</param>
         <summary>Loads an assembly given its file name or path, hash value, and hash algorithm.</summary>
         <returns>The loaded assembly.</returns>
-        <remarks>
+        <remarks FrameworkAlternate="netcore-2.0;netcore-2.1;netcore-2.2;netcore-3.0;net-5.0;net-6.0;net-7.0;net-8.0;net-9.0">
+          <format type="text/markdown"><![CDATA[
+
+## Remarks
+
+This API is present in .NET Core for compatibility, but it is recommended to use <xref:System.Reflection.AssemblyLoadContext> overloads in .NET Core 2.0 and later.
+
+The assembly is loaded into the default AssemblyLoadContext.
```

Comment:
This overload throws NotSupportedException in .NET Core 2.0+:

https://github.com/dotnet/runtime/blob/b432fc66fe59873b2b2ad584c04eaf1e4b11d611/src/libraries/System.Private.CoreLib/src/System/Reflection/Assembly.cs#L390